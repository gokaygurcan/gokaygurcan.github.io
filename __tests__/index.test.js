// gokaygurcan.github.io
//
// https://www.gokaygurcan.com/
// MIT © Gökay Gürcan

// node modules
import {shallow} from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

// local modules
import Index from '../pages/index';

describe('Enzyme', () => {
  it('Print "gokaygurcan.github.io"', () => {
    const index = shallow(
      <Index />
    );

    expect(index.find('span').text().trim()).toEqual('gokaygurcan.github.io');
  });
});

describe('Snapshot Testing', () => {
  it('Print "gokaygurcan.github.io"', () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
