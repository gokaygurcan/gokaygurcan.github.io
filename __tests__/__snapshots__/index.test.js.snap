// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing Print "gokaygurcan.github.io" 1`] = `
<div
  className="wrapper"
  data-jsx={1407449589}
>
  <header
    data-jsx={1407449589}
  >
    <nav
      data-jsx={1407449589}
    >
      <a
        data-jsx={1407449589}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
        --  
      <a
        data-jsx={1407449589}
        href="/projects"
        onClick={[Function]}
      >
        Projects
      </a>
    </nav>
  </header>
  <span>
    gokaygurcan.github.io
  </span>
  <footer
    data-jsx={1407449589}
  >
    MIT © Gökay Gürcan
  </footer>
</div>
`;
